CONFIG/AboutDlg.cpp

```diff
@@ -0,0 +1,37 @@
+#include "AboutDlg.h"
+
+#include "decomp.h"
+
+DECOMP_SIZE_ASSERT(CAboutDialog, 0x60)
+
+// FUNCTION: CONFIG 0x00403c20
+CAboutDialog::CAboutDialog() : CDialog(IDD)
+{
+}
+
+// FUNCTION: CONFIG 0x00403c90
+void CAboutDialog::BeginModalState()
+{
+	::EnableWindow(this->m_hWnd, FALSE);
```

Comment:
we still enforcing style for not using this->?

Reply:
Consider yourself an enforcer now :)
It's removed now officer ;)